include/openmc/variance_reduction.h

```diff
@@ -0,0 +1,26 @@
+#ifndef VARIANCE_REDUCTION_H
+#define VARIANCE_REDUCTION_H 1
+
+#include <map>
+#include "hdf5.h"
+
+#include "openmc/xml_interface.h"
+#include "openmc/error.h"
+
+namespace openmc {
+namespace variance_reduction {
+
+  extern bool importance_splitting;
+  extern bool weight_splitting;
+  extern std::map<int,double> importances;
+
+  void read_variance_reduction(pugi::xml_node *node);
```

Comment:
Our style guide has `*` and `&` attached to the type rather than the variable name:
```C++
void read_variance_reduction(pugi::xml_node* node);
```